### DISCUSSION TITLE
Is possible nest steps in steps?

### DISCUSSION THREAD

**Discussion by daassh** [saadeghi/daisyui#2179] [Ideas / feature requests]
Just like jenkins blueocean:
![image](https://github.com/saadeghi/daisyui/assets/13551272/1587de6c-ec35-406d-81ac-47eafeb25786)

I have try by this code
```
<ul class="steps">
  <li class="step step-primary">1</li>
    <ul class="steps steps-vertical">
    <li class="step step-primary">2-1</li>
    <li class="step step-primary">2-2</li>
  </ul>
  <li class="step step-primary">3</li>
</ul>
```

and I got this
![image](https://github.com/saadeghi/daisyui/assets/13551272/f4ad5c65-cbda-4307-82d8-d3099712c62d)

**Comment by saadeghi** [comment]
I think you should use a professional flowchart library for that.
